docs/content/en/developers/metrics.md

```diff
@@ -0,0 +1,62 @@
+---
+linkTitle: "kcp Prometheus metrics"
+weight: 1
+description: >
+Inspecting Prometheus metrics of e2e runs
+---
+
+# Inspecting Prometheus metrics for e2e runs
```

Comment:
Please name the file to roughly match with this heading. mkdocs sorts based on filename. So `inspecting-metrics-for-e2e-tests` or something like that.